CI/GitLab/.gitlab-ci.yml

```diff
@@ -0,0 +1,23 @@
+stages:
+  - build
+  - deploy
+
+cache:
+  paths:
+    - vendor/
+
+build:
+  stage: build
+  image: php:7.4
+  script:
+    - apt-get update
+    - apt-get install unzip
+    - curl -sS https://getcomposer.org/installer | php
+    - php composer.phar install --no-dev --no-progress --prefer-dist
```

Comment:
Why not use the [`composer` Docker image](https://hub.docker.com/_/composer)? That way no need to update/install everything.

Reply:
The issue with the composer image is the inability to choose your php version. With this method you can choose any php version. This is something I've had to do with my projects and so I carried over the practice in this example. 

However, in the interest of brevity I will change to the composer image.